test/extensions/filters/network/http_connection_manager/config_test.cc

```diff
@@ -3028,6 +3028,9 @@ TEST_F(HttpConnectionManagerConfigTest, HeaderValidatorConfig) {
       "@type": type.googleapis.com/envoy.extensions.filters.http.router.v3.Router
   )EOF";
 
+  TestScopedRuntime scoped_runtime;
+  scoped_runtime.mergeValues({{"envoy.reloadable_features.enable_header_validator", "true"}});
```

Comment:
comment why we set this for these tests? I think you did for the last

Reply:
Done